manifests/bosh-manifest/reference-bosh-manifest.yml

```diff
@@ -159,12 +159,17 @@ properties:
       LoadPlugin cpu
       LoadPlugin disk
       LoadPlugin entropy
-      LoadPlugin interface
       LoadPlugin load
       LoadPlugin memory
       LoadPlugin swap
       LoadPlugin uptime
 
+      LoadPlugin interface
+      <Plugin interface>
```

Comment:
Shall we add `IgnoredSelected false` to make it more explicit? It is the default anyway, s that is nto so important.

https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_interface